admin-console/src/main/webapp/src/pages/EmailTemplatePage.tsx

```diff
@@ -70,29 +70,34 @@ export const EmailTemplatePage: React.FunctionComponent = () => {
     }, []);
 
     const newTemplate = useCreateTemplate();
-    const [ templates, setTemplates ] = React.useState<Partial<Template>>();
+    const [ template, setTemplate ] = React.useState<Partial<Template>>({
+        data: defaultContentTemplate
+    });
 
     const handleChange = (value: string, event: React.FormEvent<HTMLInputElement>) => {
         const target = event.target as HTMLInputElement;
-        setTemplates(prev => ({ ...prev, [target.name]: target.value }));
+        setTemplate(prev => ({ ...prev, [target.name]: target.value }));
     };
 
     const handleCodeChange = (value: string) => {
-        setTemplates(prev => ({ ...prev, content: value }));
+        setTemplate(prev => ({ ...prev, data: value }));
     };
 
-    const handleSubmit = React.useCallback((template) => {
-        const mutate = newTemplate.mutate;
-        mutate({
-            id: template.id,
-            data: template.data,
-            name: template.name,
-            description: template.description
-        }).then(() => {
-            handleBackClick();
-        });
+    const handleSubmit = React.useCallback(() => {
+        console.log(template);
+        if (template && template.data && template.name && template?.description) {
```

Comment:
Checking for the existance of these attributes that are required. 
We should probably have something here to report the errors, but i guess is better than just loading and failing